src/Sarif/Comparers/ReportingConfigurationSortingComparer.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+/// <summary>
+/// All Comparer implementations should be replaced by auto-generated codes by JSchema as 
+/// part of EqualityComparer in a planned comprehensive solution.
+/// Tracking by issue: https://github.com/microsoft/jschema/issues/141
+/// </summary>
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    internal class ReportingConfigurationSortingComparer : IComparer<ReportingConfiguration>
```

Comment:
>ReportingConfigurationSortingComparer [](http://example.com/codeflow?start=19&length=37)

How about we call this 'ReportingConfigurationComparer'

Reply:
Updated all comparers class name to `{typename}Comparer`